geoblacklight-schema.json

```diff
@@ -0,0 +1,173 @@
+    "required": [
+        "uuid",
+        "dc_identifier_s",
+        "dc_title_s",
+        "dc_description_s",
+        "dc_rights_s",
+        "dct_provenance_s",
+        "dct_references_s",
+        "georss_box_s",
+        "layer_id_s",
+        "layer_geom_type_s",
+        "layer_modified_dt",
+        "layer_slug_s",
+        "solr_bbox",
+        "solr_geom",
+        "solr_year_i"
+    ],
+    "type": "object",
+    "properties": {
+        "uuid": { 
+            "type": "string", 
+            "description": " Unique identifier for layer that is globally unique."
+
+        },
+        "dc_identifier_s": {
+            "type": "string",
+            "description": " Unique identifier for layer. May be same as UUID but may be an alternate identifier."
+        },
+        "dc_title_s": { 
+            "type": "string", 
+            "description": " Title for the layer."
+
+        },
+        "dc_description_s": { 
+            "type": "string", 
+            "description": " Description for the layer."
+
+        },
+        "dc_rights_s": { 
+            "type": "string",
+            "enum": ["Public", "Restricted"], 
+            "description": " Access rights for the layer. Must be Public or Restricted."
+
+        },
+        "dct_provenance_s": { 
+            "type": "string", 
+            "description": " Institution who holds the layer."
+
+        },
+        "dct_references_s": { 
+            "type": "string", 
+            "description": " JSON hash for external resources, where each key is a URI for the protocol or format and the value is the URL to the resource."
+
+        },
+        "georss_box_s": { 
+            "type": "string", 
+            "description": " Bounding box as maximum values for S W N E. Example: 12.6 -119.4 19.9 84.8."
+
+        },
+        "layer_id_s": { 
+            "type": "string", 
+            "description": " The complete identifier for the layer via WMS/WFS/WCS protocol. Example: druid:vr593vj7147."
+
+        },
+        "layer_geom_type_s": { 
+            "type": "string",
+            "enum": ["Point", "Line", "Polygon", "Raster", "Scanned Map", "Paper Map", "Mixed"],
+            "description": " Geometry type for layer data."
+
+        },
+        "layer_modified_dt": { 
+            "type": "string", 
+            "description": " Last modification date for the metadata record in the Solr date/time format. Example: 2014-04-30T13:48:51Z. Note that this field is indexed as a Solr date-time field."
+
+        },
+        "layer_slug_s": { 
+            "type": "string", 
+            "description": " Unique identifier visible to the user, used for Permalinks. Example: stanford-vr593vj7147."
+
+        },
+        "solr_bbox": { 
+            "type": "string", 
+            "description": "Derived from georss_box_s. Bounding box as maximum values for W S E N. Example: 76.76 12.62 84.76 19.91. Note that this field is indexed as a Solr spatial field."
```

Comment:
I believe that we should look at modifying the name of this to `rptbbox ` or something similar. Also should be modified here: https://github.com/geoblacklight/geoblacklight-schema/blob/master/conf/schema.xml#L58

Solr has a new fieldtype named "bbox" which could cause some confusion. This would change things in GeoBlacklight  and would break things but we should make the spatial search field configurable anyways. Also we should do this the "right" way.

https://cwiki.apache.org/confluence/display/solr/Spatial+Search#SpatialSearch-BBoxField